Document 'execution_summary' response part of detection APIs (#1900)
elastic/security-docs: docs/detections/api/rules/rules-api-bulk-actions.asciidoc

               "throttle":"no_actions",
               "actions":[

               ],
               "execution_summary": {                 <1>
                 "last_execution": {
                   "date": "2022-03-23T16:06:12.787Z",
                   "status": "partial failure",
                   "status_order": 20,

>                    "message": "This rule is attempting to query data from Elasticsearch indices listed in the \"Index pattern\" section of the rule definition, however no index matching",

Reviewer: I assume this is just a sample message, but if it's possible to edit the actual message that the API returns, that would be great, too. Also make the same edits to the other APIs in the PR.
```suggestion
                   "message": "This rule is attempting to query data from Elasticsearch indices listed in the \"Index pattern\" section of the rule definition, but no matching index was found.",
```

Author: Applied your suggestion, thanks! Will also edit the actual message.

>                    "message": "This rule is attempting to query data from Elasticsearch indices listed in the \"Index pattern\" section of the rule definition, however no index matching",

Reviewer 2: ```suggestion
                   "message": "This rule attempted to query data from Elasticsearch indices listed in the \"Index pattern\" section of the rule definition, however no index matching",
```

Author: Applied your suggestion, thanks!

}
--------------------------------------------------

> <1> dev:[] `execution_summary` is in development and its schema may change in the future.

Reviewer 3: ```suggestion
<1> dev:[] `execution_summary` is under development and its schema may change.
```

Author: Applied your suggestion, thanks!
